apidoc/Titanium/Locale/Locale.yml

```diff
@@ -235,3 +251,45 @@ properties:
         sections of wikipedia for reference.
     type: String
     permission: read-only
+
+  - name: systemCountry
+    summary: Country of the current system locale, as an ISO 2-letter code.
+    description: |
+      This property can be different from `currentCountry` if the app language has been
+      overridden by calling `setLanguage()` on Android versions supporting per-app locale features.
+
+      This property holds a value such as `US` or `GB`. See the 
+      [ISO 3166-1 alpha-2](http://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements) 
+      country codes section of wikipedia for reference.
+    platforms: [android]
+    type: String
+    permission: read-only
+    since: "13.1.0"
+
+  - name: systemLanguage
```

Comment:
This should be unified via `Ti.Locale.currentLanguage`. If something is overridden, it should be reflected in the current language.